addons/web/static/src/views/list/list_renderer.js

```diff
@@ -45,6 +45,7 @@ export class ListRenderer extends Component {
             ? this.props.archInfo.creates
             : [{ description: this.env._t("Add a line") }];
 
+        this.editable = this.props.editable ? this.props.archInfo.editable : false;
```

Comment:
diff --git a/addons/web/static/src/fields/x2many_field.js b/addons/web/static/src/fields/x2many_field.js
index 544b148ff78..628b82f95c6 100644
--- a/addons/web/static/src/fields/x2many_field.js
+++ b/addons/web/static/src/fields/x2many_field.js
@@ -37,10 +37,7 @@ export class X2ManyField extends Component {
         const archInfo = this.fieldInfo.views[this.viewMode];
         return {
             activeActions: this.activeActions,
-            archInfo: {
-                ...archInfo,
-                editable: this.props.record.isInEdition && archInfo.editable,
-            },
+            editable: this.props.record.isInEdition && archInfo.editable,
             fields: Object.assign({}, this.props.fields, archInfo.fields), // WOWL is this necessary?
             list: this.list,
             openRecord: this.openRecord.bind(this),
diff --git a/addons/web/static/src/views/list/list_renderer.js b/addons/web/static/src/views/list/list_renderer.js
index d3601ed647a..e4313380c8a 100644
--- a/addons/web/static/src/views/list/list_renderer.js
+++ b/addons/web/static/src/views/list/list_renderer.js
@@ -400,7 +400,7 @@ export class ListRenderer extends Component {
     }
 
     async onCellClicked(record, column) {
-        if (this.props.archInfo.editable) {
+        if (this.props.editable) {
             if (record.isInEdition) {
                 this.focusCell(column);
                 this.cellToFocus = null;